develop/tutorials/articles/02-from-liferay-6-to-liferay-7/07-upgrading-plugins-to-liferay-7/03-upgrading-the-liferay-maven-environment.markdown

```diff
@@ -0,0 +1,107 @@
+
+## Upgrading Liferay Maven Plugins
+
+The biggest change for your project's build plugins is the removal of the
+`liferay-maven-plugin`. Liferay now provides several individual Maven plugins
+that accomplish specific tasks. For example, you can configure Maven plugins for
+Liferay's CSS Builder, Service Builder, Theme Builder, etc. With smaller plugins
+available to accomplish specific tasks in your project, you no longer have to
+rely on one bulky plugin that provides many things you may not want.
+
+As an example, suppose your Liferay Portal 6.2 project was using the
+`liferay-maven-plugin` to only use Liferay CSS Builder. First, you should remove
+the legacy `liferay-maven-plugin` plugin dependency from your project's
+`pom.xml` file:
+
+    <plugin>
+        <groupId>com.liferay.maven.plugins</groupId>
+        <artifactId>liferay-maven-plugin</artifactId>
+        <version>${liferay.version}</version>
+        <configuration>
+            <autoDeployDir>${liferay.auto.deploy.dir}</autoDeployDir>
+            <liferayVersion>${liferay.version}</liferayVersion>
+            <pluginType>portlet</pluginType>
+        </configuration>
+    </plugin>
+
+Then, add the CSS Builder plugin dependency to your project's `pom.xml` file:
+
+    <plugin>
+        <groupId>com.liferay</groupId>
+        <artifactId>com.liferay.css.builder</artifactId>
+        <version>1.0.21</version>
+        <executions>
+            <execution>
+                <id>default-build-css</id>
+                <phase>generate-sources</phase>
+                <goals>
+                    <goal>build-css</goal>
+                </goals>
+            </execution>
+        </executions>
+            <configuration>
+                <docrootDirName>src/main/webapp</docrootDirName>
+            </configuration>
+    </plugin>
+
+Some popular Liferay Maven plugins are listed below, with their corresponding
+artifact IDs and tutorials explaining how to configure them:
+
+**Popular Liferay Maven Plugins**
+
+Name | Artifact ID | Tutorial |
+:----| :---------- | :------- |
+CSS Builder | [com.liferay.css.builder](https://search.maven.org/#search%7Cga%7C1%7Ccom.liferay.css.builder) | [Compiling SASS Files in a Maven Project](/develop/tutorials/-/knowledge_base/7-0/compiling-sass-files-in-a-maven-project) |
+Lang Builder | [com.liferay.lang.builder](https://search.maven.org/#search%7Cga%7C1%7Ccom.liferay.lang.builder) | Coming Soon |
+Service Builder | [com.liferay.portal.tools.service.builder](https://search.maven.org/#search%7Cga%7C1%7Ccom.liferay.portal.tools.service.builder) | [Using Service Builder in a Maven Project](/develop/tutorials/-/knowledge_base/7-0/using-service-builder-in-a-maven-project) |
+Theme Builder | [com.liferay.portal.tools.theme.builder](https://search.maven.org/#search%7Cga%7C1%7Ccom.liferay.portal.tools.theme.builder) | [Building Themes in a Maven Project](/develop/tutorials/-/knowledge_base/7-0/building-themes-in-a-maven-project) |
```

Comment:
@david-truong I listed out four Maven Plugins that we're using for Liferay 7 development (for traditional plugins) for devs that are making the switch from Liferay 6 -> 7:

- CSS Builder
- Lang Builder
- Service Builder
- Theme Builder

Are there any others I should highlight, or would be useful for devs using Maven?